pkgpanda/docs/package_concepts.md

```diff
@@ -39,13 +39,13 @@ starting the service.
 
 *Well-known directories*
 
-Every pkgpanda package may put items in several well-known directories to have them available to other packages. Additionally, these directories will be sym-linked into system environment paths upon package activation.
+Every pkgpanda package may put items in several well-known directories within the package to have them available to other packages globally via symlinks.
 
 ```bash
-lib/  # Will be linked into $LD_LIBRARY_PATH
-bin/  # Will be linked into $PATH
-etc/  # Will be linked into system's /etc
-dcos.target.wants/  # Will be linked into /etc/systemd/system/dcos.target.wants
+lib/  # Will be linked to /opt/mesosphere/lib
+bin/  # Will be linked to /opt/mesosphere/bin
+etc/  # Will be linked to /opt/mesosphere/etc
+dcos.target.wants/  # Will be linked into /etc/systemd/system/dcos.target.wants depending on role
```

Comment:
This "depending on role" applies to all well-known dirs. For example, you can add a config file intended just for masters by putting it in `$PKG_PATH/etc_master/`.